cmd/podman/shared/container.go

```diff
@@ -88,6 +88,7 @@ type PsContainerOutput struct {
 	PIDNS     string
 	User      string
 	UTS       string
+	Mounts    string
```

Comment:
The nitpicky side of me says we should probably omit this from the ps JSON output, but I don't really care enough to hold the PR.